DigitalOffice/Tests/FileServiceTests/Commands/GetFileByIdCommandTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using LT.DigitalOffice.FileService.Commands;
+using LT.DigitalOffice.FileService.Commands.Interfaces;
+using LT.DigitalOffice.FileService.Database.Entities;
+using LT.DigitalOffice.FileService.Mappers.Interfaces;
+using LT.DigitalOffice.FileService.Models;
+using LT.DigitalOffice.FileService.Repositories.Interfaces;
+using LT.DigitalOffice.FileServiceUnitTests.UnitTestLibrary;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.FileServiceUnitTests.Commands
+{
+    public class GetFileByIdCommandTests
+    {
+        private IGetFileByIdCommand command;
+        private Mock<IFileRepository> repositoryMock;
+        private Mock<IMapper<DbFile, File>> mapperMock;
+
+        private DbFile file;
```

Comment:
We do not separate fields from each other

Reply:
Fixed